tests/EdFi.Ods.Api.IntegrationTests/ManualPostgreSqlTests.cs

```diff
@@ -43,6 +43,16 @@ protected override IDbConnection BuildTestConnection()
             }
         }
 
+        [TestFixture]
+        [Explicit]
+        public class PostgreSqlStaffUSIToEducationOrganizationIdAuthViewTests : StaffUSIToEducationOrganizationIdAuthViewTests
+        {
```

Comment:
Could you please change it from 'StaffUSI' to 'StaffUsi' in all places ? basically 'USI' to 'Usi' including file name